### PR TITLE
ipconfig: add German translation

### PR DESCRIPTION
**Pull request by Xuki**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution.

File: pages.de/windows/ipconfig.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# ipconfig
+
+> Zeigt die Netzwerkkonfiguration von Windows an und verwaltet diese.
```
Comment (gutjuri) [inline pages.de/windows/ipconfig.md:3]:
```suggestion
> Zeige die Netzwerkkonfiguration von Windows an und verwalte diese.
```

File: pages.de/windows/ipconfig.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+> Weitere Informationen: <https://learn.microsoft.com/windows-server/administration/windows-commands/ipconfig>.
+
+- Zeigt eine Liste der Netzwerkadapter an:
```
Comment (gutjuri) [inline pages.de/windows/ipconfig.md:6]:
```suggestion
- Zeige eine Liste der Netzwerkadapter an:
```

File: pages.de/windows/ipconfig.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`ipconfig`
+
+- Zeigt eine detaillierte Liste der Netzwerkadapter an:
```
Comment (gutjuri) [inline pages.de/windows/ipconfig.md:10]:
```suggestion
- Zeige eine detaillierte Liste der Netzwerkadapter an:
```

File: pages.de/windows/ipconfig.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`ipconfig /all`
+
+- Erneuert die IP-Adressen für einen Netzwerk Adapter:
```
Comment (gutjuri) [inline pages.de/windows/ipconfig.md:14]:
```suggestion
- Erneuere die IP-Adressen für einen Netzwerkadapter:
```

File: pages.de/windows/ipconfig.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`ipconfig /renew {{Adapter}}`
+
+- Gibt die IP-Adressen für einen Netzwerkadapter frei:
```
Comment (gutjuri) [inline pages.de/windows/ipconfig.md:18]:
```suggestion
- Gib die IP-Adressen für einen Netzwerkadapter frei:
```

File: pages.de/windows/ipconfig.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`ipconfig /release {{Adapter}}`
+
+- Zeigt den lokalen DNS-Cache an:
```
Comment (gutjuri) [inline pages.de/windows/ipconfig.md:22]:
```suggestion
- Zeige den lokalen DNS-Cache an:
```

File: pages.de/windows/ipconfig.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+`ipconfig /displaydns`
+
+- Entfernt alle Daten aus dem lokalen DNS-Cache:
```
Comment (gutjuri) [inline pages.de/windows/ipconfig.md:26]:
```suggestion
- Entferne alle Einträge aus dem lokalen DNS-Cache:
```

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thanks! We prefer to use the imperative form in the descriptions. Apart from that, LGTM :)

**Comment by Xuki** [comment]
Changed the descriptions to imperative form, thanks for letting me know :)

**Review by kbdharun — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]
Thanks!

**Pull request merged**